docs/runtime_environment.rst

```diff
@@ -56,3 +56,11 @@ Note that non-relative imports to these files will `not` work:
 
    import backend
    from utils import img_utils, ml_utils
+
+Limiting GPU memory Growth
+-----------------------------------------
+
+By default Vcap maps all available memory of all visible CUDA configured GPUs,
```

Comment:
I think this section could make it a bit more clear that this advice only applies when using a TensorFlow model.